frontend/src/utils/locale.js

```diff
@@ -20,6 +20,20 @@ export function getCurrentActiveLocale() {
   return localStorage.getItem(LOCAL_LANG);
 }
 
+/**
+ * @summary extracts the language from the locale
+ * @param {string} locale in form of 'en_US', 'de_DE
+ * @returns {string}
```

Comment:
best practices for writting javadoc / jsdoc:

> @summary extracts the language from the locale

DRY, basically that's exactly what the function name says too.
And that's great, because a good name which is intention revealing does not need to write a function summary.

> @returns {string}

Basically here you need to write how it returns, e.g.
```
@returns {string} extracted language (e.g. 'en', 'de' etc)
```